### ISSUE TITLE
Feature: Video Rename

### ISSUE THREAD

**Issue by lafemmephile** [lafemmephile/core#41]
# Purpose
Sometimes it is useful to *rename* a file (e.g. for *randomization* purposes) or multiple files.

# CLI
Potential CLI design

## Arguments/Options
+ `video_path`: required argument for path to video file(s)
+ `out_path`: required argument for path to move video file(s) to.
+ `-a/--append`: click.Choice option to simply append new random name to existing name (default `None`)
+ `-r/--random`: boolean option to turn on random name generation (default `False`)

## Signature
```console
femmephile rename -rk -p rndm_smpl -s comp_001 <video_path> <out_path>
```

**Comment by lafemmephile** [comment]
# References
+ https://docs.python.org/3/library/shutil.html#shutil.move

**Comment by lafemmephile** [comment]
Implemented with: ff56ca7789a73804741188908f9c961dbce991c7
